src/core_plugins/tile_map/public/coordinatemap_response_handler.js

```diff
@@ -0,0 +1,25 @@
+import { tabifyAggResponse } from 'ui/agg_response/tabify/tabify';
+import { convertToGeoJson } from 'ui/vis/map/convert_to_geojson';
+
+export function makeTabbedResponseHandler() {
+
+  let lastEsResponse;
+  let lastGeoJsonResponse;
+
+  return function makeIdentityResponseHandler(vis, esResponse) {
```

Comment:
Looks like this now modifies the response. Should this be renamed to geoJson for something like that?

Reply:
yes, needs rename indeed